lk2nd/device/dts/msm8952/sdm429-lenovo-tbx505x.dts

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
```

We try to follow linux license preference for dts, I'd appreciate if you relicense it.